tests/queries/tests.py

```diff
@@ -96,9 +95,9 @@ def test_explain(self):
         supported_formats = connection.features.supported_explain_formats
         all_formats = (None,) + tuple(supported_formats) + tuple(f.lower() for f in supported_formats)
         for idx, queryset in enumerate(querysets):
-            for verbose, output_format in itertools.product((True, False), all_formats):
-                with self.subTest(verbose=verbose, format=output_format, queryset=idx):
-                    result = queryset.explain(format=output_format, verbose=verbose)
```

Comment:
As the `verbose` argument is removed here, add it to `test_kwargs` in `test_postgres_explain_kwargs()`?

While you're there, also add the `timing` option.

There is also `summary` for PostgreSQL 10+ only, so that could be added with something along the lines of:

```python
if connection.pg_version >= 100000:
    test_kwargs.append({'summary': True})
```